syntax = "proto3";

package nodesidecar;

import public "nucliadb_protos/noderesources.proto";

message Counter {
    uint64 fields = 1;
    uint64 paragraphs = 2;
}


service NodeSidecar {
    rpc GetCount(noderesources.ShardId) returns (Counter) {}
}
